x-pack/plugins/apm/server/lib/apm_telemetry/collect_data_telemetry/tasks.ts

```diff
@@ -626,8 +626,39 @@ export const tasks: TelemetryTask[] = [
         Promise.resolve({} as Record<AgentName, number>)
       );
 
+      const services = await telemetryClient.search({
+        index: [
+          indices.error,
+          indices.span,
+          indices.metric,
+          indices.transaction,
+        ],
+        terminate_after: 1,
+        body: {
+          size: 0,
+          track_total_hits: false,
+          query: {
+            query: {
```

Comment:
typo, query should not be present twice

Reply:
good catch, fixed https://github.com/elastic/kibana/pull/166925/commits/81b2257bf588da66892b8596f772ff671005b7b0